pubsub/spec/quickstart_spec.rb

```diff
@@ -13,22 +13,21 @@
 # limitations under the License.
 
 require "rspec"
-require "google/cloud"
+require "google/cloud/pubsub"
 
 describe "PubSub Quickstart" do
 
   it "creates a new topic" do
-    gcloud = Google::Cloud.new ENV["GOOGLE_CLOUD_PROJECT"]
-    pubsub = gcloud.pubsub
+    pubsub = Google::Cloud::Pubsub.new
```

Comment:
Project ID is implied from GOOGLE_CLOUD_PROJECT environment variable.
Project ID does not have to explicitly set in tests.
Project ID is explicitly set in code snippets until project detection is added (google-cloud-ruby [#951](https://github.com/GoogleCloudPlatform/google-cloud-ruby/issues/951))